CONTRIBUTING.md

```diff
@@ -1,69 +1,216 @@
-guide on writing and formatting on GitHub as a starting point.
-
-ROCm documentation adds additional requirements to Markdown and RST based files
-as follows:
-
-* Level one headers are only used for page titles. There must be only one level
-  1 header per file for both Markdown and Restructured Text.
-* Pass [markdownlint](https://github.com/markdownlint/markdownlint) check via
-  our automated GitHub action on a pull request (PR).
-  See the {doc}`rocm-docs-core linting user guide <rocm-docs-core:user_guide/linting>` for more details.
-
-## Filenames and folder structure
-
-Please use kebab-case (all lower case letters and dashes instead of spaces)
-for file names. For example, `example-file-name.md`.
-Our documentation follows Pitchfork for folder structure.
-All documentation is in `/docs` except for special files like
-the contributing guide in the `/` folder. All images used in the documentation are
-placed in the `/docs/data` folder.
+AMD values and encourages contributions to our code and documentation. If you choose to
+contribute, we encourage you to be polite and respectful. Improving documentation is a long-term
+process, to which we are dedicated.
 
-## Language and style
+If you have issues when trying to contribute, refer to the
+[discussions](https://github.com/RadeonOpenCompute/ROCm/discussions) page in our GitHub
+repository.
+
+## Folder structure and naming convention
+
+Our documentation follows the Pitchfork folder structure. Most documentation files are stored in the
+`/docs` folder. Some special files (such as release, contributing, and changelog) are stored in the root
+(`/`) folder.
+
+All images are stored in the `/docs/data` folder. An image's file path mirrors that of the documentation
+file where it is used.
+
+Our naming structure uses kebab case; for example, `my-file-name.rst`.
+
+## Supported formats and syntax
+
+Our documentation includes both Markdown and ReST files. We are gradually transitioning existing
+Markdown to ReST in order to more effectively meet our documentation needs. When contributing,
+ReST is preferred; if you must use Markdown, use GitHub-flavored Markdown.
+
+We use [Sphinx Design](https://sphinx-design.readthedocs.io/en/latest/index.html) syntax and compile
+our API references using Doxygen.
+
+The following table shows some common documentation components and the syntax convention we
+use for each:
+
+<table>
+<tr>
+<th>Component</th>
+<th>ReST syntax</th>
+</tr>
+<tr>
+<td>Code blocks</td>
+<td>
+
+```rst
+
+..  code-block:: language-name
+
+My code block.
+
+
+```
+
+</td>
+</tr>
+<tr>
+<td>Cross-referencing internal files</td>
+<td>
+
+```rst
+
+:doc:`Title <../path/to/file/filename>`
+
+```
+
+</td>
+</tr>
+<tr>
+<td>External links</td>
+<td>
+
+```rst
+
+`link name  <URL>`_
+
+```
+
+</td>
+</tr>
+<tr>
+<tr>
+<td>Headings</td>
+<td>
+
+```rst
+
+******************
+Chapter title
```

Comment:
Do we want to put the heading equivalents here for clarity? For example,

Chapter title <H1>
Section title <H2>
Subsection title <H3>
Sub-subsection title <H4>